File: Bucles/Ejercicio12-Bucles.cpp
/*12. Hacer un programa que calcule el resultado de la siguiente expresión: 1-2+3-4+5-6...n*/

#include<iostream>
#include<stdlib.h>

using namespace std;

int main(){
    int numero, resultado = 0;
    cout<<"Digite numero de elementos: "; cin>>numero;
    
    for (int i=1 ;i<=numero; i++){
        if (i%2 == 0) {
            resultado -= i;
        }
        else{
            resultado += i;
        }
    }

    cout<<"\nEl resultado es: "<<resultado<<endl;

    system("pause");
    return 0;
}

File: Comparativa Python/Ejercicio12-Bucles.py
""" 12. Hacer un programa que calcule el resultado de la siguiente expresión:
1-2+3-4+5-6...n"""

n = int(input('Digite valor a iterar: '))
sumatoria = 0

for i in range(1, n+1, 1):
    if n % 2 == 0:
        sumatoria -= i
    else:
        sumatoria += i
    
    print('{} = {}'.format(i, sumatoria))

print('El resultado de la sumatoria es: ', sumatoria)
